commands/watch.js

```diff
@@ -38,7 +38,6 @@ var run = module.exports.run = function(opts) {
         meta: opts.meta
       }, function(err, res) {
         if(err) {
-          console.error(err);
```

Comment:
@bruce-lindsay I must have removed that some time ago.  I think the double error was annoying me - however, I have no issue with reverting that change.